assets/dev/js/editor/elements/views/widget.js

```diff
@@ -1,39 +1,22 @@
 import WidgetDraggable from './behaviors/widget-draggable';
 import WidgetResizable from './behaviors/widget-resizeable';
+import BaseWidget from './base-widget';
 
-var BaseElementView = require( 'elementor-elements/views/base' ),
-	WidgetView;
-
-WidgetView = BaseElementView.extend( {
+const WidgetView = BaseWidget.extend( {
```

Comment:
For better GIT history management, maybe rename the current file to `base-widget` and create another file for the `widget`.

Reply:
Handled.

Reply:
Handled.